packages/api-client/index.ts

```diff
@@ -11,13 +20,94 @@ class APIClient {
     index: async (): Promise<GetClubResponse> => {
       return (await this.axios.get("/api/club")).data;
     },
-    create: async (p: CreateClubParams): Promise<CreateClubResponse> => {
-      return (await this.axios.post("/api/club", p)).data;
+    create: async (params: CreateClubParams): Promise<CreateClubResponse> => {
+      return (await this.axios.post("/api/club", params)).data;
+    },
+  };
+  profile = {
+    index: async (): Promise<GetProfileResponse> => {
+      return (await this.axios.get(`/v1/profile`)).data;
+    },
+  };
+  course = {
+    get: async (courseId: number): Promise<GetCourseResponse> => {
+      const course = (await this.axios.get(`/api/v1/courses/${courseId}`)).data;
+      course.officeHours.forEach((officeHour: any) =>
+        parseOfficeHourDates(officeHour)
+      );
+      return course;
+    },
+  };
+  taStatus = {
+    update: async (courseId: number): Promise<TAUpdateStatusResponse> => {
+      const queue = (
+        await this.axios.patch(`/v1/courses/${courseId}/ta/change_status`)
+      ).data;
+      parseQueueDates(queue);
+      queue.questions.forEach((question: any) => parseQuestionDates(question));
+      return queue;
+    },
+  };
+  questions = {
```

Comment:
we're gonna have to prepend each of these endpoints with `/api` in the future, since frontend routing is all being done automatically by Next, prepending with `/api` makes our routing not bugger out